File: dia1/prueba.py

#Sucession Fibonacci (0 1 1 2 3 ....)

def fibonacci(n):
    resultado = []
    a, b = 0, 1
    while (a <= n):
        resultado.append(a)
        a, b = b, a + b
    return resultado

print(fibonacci(1000))

File: dia1/prueba.js

//Sucesion fibonacci

function fibonacci(n){
    resultado=[0,1]
    for(var i=2;i<=n;i++){
        var a=resultado[i-1]
        var b=resultado[i-2]
        resultado.push(a+b)

    }
    return resultado
}

console.log(fibonacci(10))
